biodatasets/epicqa/epicqa.py

```diff
@@ -0,0 +1,295 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+
+import datasets
+
+from utils import schemas
+from utils.constants import Tasks
+from utils.configs import BigBioConfig
+
+_CITATION = """\
+Unknown
+"""
+
+_DESCRIPTION = """\
+In response to the COVID-19 pandemic, the Epidemic Question
+Answering (EPIC-QA) track challenges teams to develop systems
+capable of automatically answering ad-hoc questions about the
+disease COVID-19, its causal virus SARS-CoV-2, related corona
+viruses, and the recommended response to the pandemic.
+"""
```

Comment:
can we also include some quotes (maybe all) of the language that describes the two tasks ? 
```
The 2020 EPIC-QA track involves two tasks:

Task A
    Expert QA: In Task A, teams are provided with a set of questions asked by experts and are asked to provide a ranked list of expert-level answers to each question. In Task A, answers should provide information that is useful to researchers, scientists, or clinicians.

Task B
    Consumer QA: In Task B, teams are provided with a set of questions asked by consumers and are asked to provide a ranked list of consumer-friendly answers to each question. In Task B, answers should be understandable by the general public.

While each task will have its own set of questions, many of the questions will overlap. This is by design, so that the collection can be used to explore whether the same approaches or systems can account for different types of users.
```   
https://bionlp.nlm.nih.gov/epic_qa/#collection